packages/EasyApiToken/tests/AbstractFirebaseJwtTokenTestCase.php

```diff
@@ -43,7 +43,7 @@ protected function createFirebaseJwtDriver(
         OpenSSLAsymmetricKey|string|null $publicKey = null,
         OpenSSLAsymmetricKey|string|null $privateKey = null,
         ?array $allowedAlgos = null,
-        ?int $leeway = null
+        ?int $leeway = null,
```

Comment:
We never had trailing comas on function arguments before

Reply:
Thanks. Fixed